tachyon/zk/plonk/keys/key.h

```diff
@@ -93,7 +93,7 @@ class TACHYON_EXPORT Key {
 
     result->fixed_columns =
         base::Map(assembly.fixed_columns(), [](const RationalEvals& evals) {
-          std::vector<F> result(evals.evaluations().size());
+          std::vector<F> result(evals.evaluations().size(), F::Zero());
```

Comment:
It seems like we don't need to initialize the values, since `BatchEvaluate()` will be filled with the values. But just add a comment that why this is safe.